lib/clearance/authorization.rb

```diff
@@ -19,7 +19,7 @@ module Authorization
     #     end
     def require_login
       unless signed_in?
-        deny_access
+        deny_access(I18n.t('flashes.failure_when_access_denied'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.